cogs/system.py

```diff
@@ -2,9 +2,187 @@
+        for i, cogs in enumerate(self.selects):
+            self.selects[i].reload = not self.selects[i].reload
+
+        if self.selects[0].reload:
+            button.style = disnake.ButtonStyle.green
+            button.label = "Reload on"
+        else:
+            button.style = disnake.ButtonStyle.secondary
+            button.label = "Reload off"
+
+        await inter.response.edit_message(view=self)
+
+    async def on_timeout(self):
+        lenght = len(self.children)
+        for x in range(lenght):
+            self.remove_item(self.children[0])
+        await self.message.edit(view=self)
+
+
+class Dropdown(disnake.ui.Select):
+    def __init__(self, bot, view, cogs):
+        self.bot = bot
+        self._view = view
+        self.cogs = cogs
+        self.reload = False
+        self.msg = ""
+        self.unloadable_cogs = ["system"]
+
+        super().__init__(
+            placeholder=self.get_initials(),
+            min_values=1,
+            max_values=len(self.cogs[0]),
+            options= self.create_select()
+        )
+
+    def get_initials(self):
+        """Creates placeholder for selects from names of cogs."""
+        first = self.cogs[0][0].capitalize()
+        last = self.cogs[0][len(self.cogs)-1].capitalize()
+        return f"{first} - {last}"
+
+    def create_select(self):
+        """Creates one singular select from cogs"""
+        options = []
+        dict_of_cogs = dict(zip(self.cogs[0], self.cogs[1]))
+        val_list = list(dict_of_cogs.values())
+        key_list = list(dict_of_cogs.keys())
+
+        loaded = []
+        for value in val_list:
+            if value in self.bot.cogs:
+                position = val_list.index(value)
+                loaded.append(key_list[position])
+
+        for cog in dict_of_cogs:
+            cogname = cog.capitalize()
+            if cog.lower() in loaded:
+                options.append(disnake.SelectOption(label=cogname, value=cog, emoji="✅"))
+            else:
+                options.append(disnake.SelectOption(label=cogname, value=cog, emoji="❌"))
+        return options
+
+    def create_cog_lists(self):
+        cog_files = utils.get_all_cogs().keys()
+
+        # list out keys and values separately
+        key_list = self.cogs[0]
+        val_list = self.cogs[1]
+        bot_cogs = [cog for cog in self.bot.cogs]
+
+        loaded = []
+        for value in val_list:
+            if value in bot_cogs:
+                position = val_list.index(value)
+                loaded.append(key_list[position])
+
+        unloaded = list(set([cog.lower() for cog in cog_files]) - set(loaded))
+        loaded.sort()
+        unloaded.sort()
+        return (loaded, unloaded)
+
+    def create_embed(self):
+        embed = disnake.Embed(title="Cogs information and loading", colour=disnake.Colour.red())
+
+        key_list = list(utils.get_all_cogs().keys())
+        val_list = list(utils.get_all_cogs().values())
```

Comment:
you are listing the files twice here, once should be enough
```suggestion
        all_cogs = utils.get_all_cogs()
        key_list = list(all_cogs.keys())
        val_list = list(all_cogs.values())
```
Maybe we can better name those variables, one of them contains class names and the other file names right?